src/Index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import RoutedApp from './RoutedApp';
+import siteConfig from './config/site.config';
+
+// eslint-disable-next-line no-undef
```

Comment:
Add inline comment mentioning where `SITE_CONFIG` comes from. This global variable was defined in the [site.webpack.config](https://github.com/cerner/terra-site/blob/f9d19943820dc74d967a3023c4c1cb0c5bdc0bae/scripts/start-terra-site/site.webpack.config.js#L31) by the DefinePlugin.